lang/rust/avro/src/schema.rs

```diff
@@ -1282,6 +1280,23 @@ impl Parser {
         Ok(schema)
     }
 
+    fn validate_name(
+        &self,
+        name: &String
+    ) -> bool {
+        let i1 = name.len();
+        if i1 == 0 {
```

Comment:
```suggestion
        if §name.len() == 0 {
```
`i1` is a bad name anyway